commandFunctions/misc.js

```diff
@@ -85,9 +85,9 @@ module.exports =
 	},
 	chucknorris: function(arg, msg)
 	{
-	url = `https://api.chucknorris.io/jokes/random`;
+	const url = `https://api.chucknorris.io/jokes/random`;
 
-	footer = `Requested by: ${msg.author.tag} | Provided by api.chucknorris.io`;
+	var footer = `Requested by: ${msg.author.tag} | Provided by api.chucknorris.io`;
```

Comment:
Why are you using var? You only define it once. It's clearly a const. Also to add: If you'd use var in this context because you reassign it I'd use let. Usually better. If you don't know the difference read up on the docs.

Reply:
Because Javascript is hard.